src/biogeochem/CNFireBaseMod.F90

```diff
@@ -249,13 +253,18 @@ subroutine CNFireInterp(this,bounds)
     ! !DESCRIPTION:
     ! Interpolate CN Fire datasets
     !
+    ! !USES:
+    use clm_varctl, only: use_cn
+    !
     ! !ARGUMENTS:
     class(cnfire_base_type) :: this
     type(bounds_type), intent(in) :: bounds  
     !-----------------------------------------------------------------------
 
     if ( this%need_lightning_and_popdens ) then
-       call this%hdm_interp(bounds)
+       if (use_cn) then
+          call this%hdm_interp(bounds)
+       end if
```

Comment:
@jkshuman this is another `if (use_cn)` that would need to be removed to allow SPITFIRE access to the human pop. density dataset.